MREUnityRuntime/MREUnityRuntimeLib/Patching/Types/PhysicsBridgePatch.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using MixedRealityExtension.Animation;
+using MixedRealityExtension.Core.Physics;
+using Newtonsoft.Json.Linq;
+
+namespace MixedRealityExtension.Patching.Types
+{
+	public class TransformPatchInfo
+	{
+		public TransformPatchInfo() { }
+
+		internal TransformPatchInfo(Guid id, RigidBodyTransform transform)
+		{
+			Id = id;
+
+			Transform = new TransformPatch();
+			Transform.Position = new Vector3Patch(transform.Position);
+			Transform.Rotation = new QuaternionPatch(transform.Rotation);
+		}
+
+		/// <summary>
+		/// ID of the actor (of the RB)
+		/// </summary>
+		public Guid Id { get; set; }
+
+		public TransformPatch Transform { get; set; }
+	}
+
+	public class PhysicsBridgePatch : IPatchable
+	{
+		public PhysicsBridgePatch()
+		{
+			TransformCount = 0;
+			TransformsBLOB = null;
+		}
+
+		internal PhysicsBridgePatch(Guid sourceId, Snapshot snapshot)
+		{
+			Id = sourceId;
+			Time = snapshot.Time;
+
+			TransformCount = snapshot.Transforms.Count;
+
+			if (TransformCount > 0)
+			{
+				// copy transform to native array to reinterpret it to byte array without 'unsafe' code
+				// todo: we should use native array in snapshot anyway.
+				Unity.Collections.NativeArray<Snapshot.TransformInfo> transforms =
+						new Unity.Collections.NativeArray<Snapshot.TransformInfo>(snapshot.Transforms.ToArray(), Unity.Collections.Allocator.Temp);
+
+				int sizeOfTransformInfo = Unity.Collections.LowLevel.Unsafe.UnsafeUtility.SizeOf<Snapshot.TransformInfo>();
+				TransformsBLOB = transforms.Reinterpret<byte>(sizeOfTransformInfo).ToArray();
```

Comment:
As you mentioned offline, this API isn't available in 2019.2 😢.

Reply:
this is fixed in #164